dev/releases/update_website.py

```diff
@@ -80,21 +80,21 @@ def download_asset_by_name(asset_name: str, writedir: str) -> None:
         utils.download_with_sha256(url, asset_name)
 
 
-def extract_tarball(tarball: str) -> None:
-    notice(f"Extracting {tarball} . . .")
-    with tarfile.open(tarball) as tar:
+def extract_tarball(tball: str) -> None:
+    notice(f"Extracting {tball} . . .")
+    with tarfile.open(tball) as tar:
         try:
             tar.extractall()
         except:
-            error(f"Failed to extract {tarball}!")
+            error(f"Failed to extract {tball}!")
 
 
 def get_date_from_configure_ac(gaproot: str) -> str:
-    with open(f"{gaproot}/configure.ac", "r") as configure_ac:
+    with open(f"{gaproot}/configure.ac", "r", encoding="utf-8") as configure_ac:
         filedata = configure_ac.read()
         try:  # Expect date in YYYY-MM-DD format
             release_date = re.search(
-                "\[gap_releaseday\], \[(\d{4}-\d{2}-\d{2})\]", filedata
+                r"\[gap_releaseday\], \[(\d{4}-\d{2}-\d{2})\]", filedata
```

Comment:
Huh. Was this broken before then? Or is this just recommended but it works the same anyway?

If this changes behavior we should check what the effect is, if any.